chain-cli/src/commands/deploy/index.spec.ts

```diff
@@ -58,7 +58,7 @@ describe("Deploy Command", () => {
     expect(result.join()).toContain(
       "Deployment scheduled to sandbox. Status CC_DEPLOY_SCHEDULED for Chaincode chaincode-name and Channel channel-name."
     );
-  });
+  }, 10000);
```

Comment:
actually in most cases we use `jest.setTimeout` instead of passing it for a single test